### PR TITLE
Meetups/scroll view

### PR DESCRIPTION
**Pull request by atappy**
In this Sprint I started by trying to understand how activities and views work in android studio.

Then I created a custom scrollView to contain list of meetups, as a linearLayout. This linearLayout will contain every meetup available, each of them contained in an holder (named meetup_listview). 

Then I created a class for my MeetupList, that takes from another activity (using extra) a list of Meetups, and then process these informations and add them at the correct place in the corresponding meetup_listview.

### CODE REVIEW

File: app/src/androidTest/java/com/github/palFinderTeam/palfinder/MainActivityTest.kt [hunk 1]
```diff
@@ -13,25 +13,3 @@ import org.junit.Rule
...
-            .perform(ViewActions.closeSoftKeyboard())
-
-        onView(ViewMatchers.withId(R.id.mainGoButton))
-            .perform(ViewActions.click())
-
-        intended(IntentMatchers.hasExtra(EXTRA_MESSAGE, message))
-        Intents.release()
-        }
-
-}
```
Comment (pgilliar) [inline app/src/androidTest/java/com/github/palFinderTeam/palfinder/MainActivityTest.kt:37]: maybe keep a test to check if you went to your activity even if it's the old button

**Review by TheblueMan003 — APPROVED** [review]
There are some Hard-Coded Strings but otherwise good for now.

**Review by pgilliar — APPROVED** [review]

File: app/src/main/java/com/github/palFinderTeam/palfinder/MeetupListActivity.kt [hunk 2]
```diff
@@ -0,0 +1,82 @@
...
+    }
+
+    @RequiresApi(Build.VERSION_CODES.N)
+    private fun timeDiff(start: android.icu.util.Calendar, end: Calendar) : String {
+        val diff: Long = end.timeInMillis - start.timeInMillis
+        val seconds = diff / 1000
+        val minutes = seconds / 60
+        val hours = minutes / 60
+        if (hours < 24) {
+            return "commence dans " + hours + " heures"
```
[the remark this answers is not in the document]
Comment (atappy) [reply app/src/main/java/com/github/palFinderTeam/palfinder/MeetupListActivity.kt:60]: As Daniel has already made a class adressing the date problem better than I did, I think we're not gonna keep this function, that was just sort of a test to populate the MeetupListView

File: app/src/androidTest/java/com/github/palFinderTeam/palfinder/MainActivityTest.kt (hunk 1 above)
Comment (atappy) [reply app/src/androidTest/java/com/github/palFinderTeam/palfinder/MainActivityTest.kt:37]: I think I am going to do a better test for it when we merge everything together for the demo, but thank you for the comment !

**Pull request merged**